File: FixedSlidingWindow/Java/FixedSlidingWindow.java
public class FixedSlidingWindow {
    public static void main(String[] args) {

        int arr[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};

        int start = 0; 
        int windowSize = 3;
        int windowSum = 0;
        
        for(int end = 0; end < arr.length; end++) {

            windowSum += arr[end];
            
            // window size exceeded
            if(end >= windowSize) {
                windowSum -= arr[start];
                start++;
            } 

            System.out.println("Window sum now is " + windowSum);
        }
    }
}

File: FixedSlidingWindow/Python/FixedSlidingWindow.py

arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

windowSize = 3
windowSum = 0
start = 0

for end in range(len(arr)):

    windowSum += arr[end]

    # window exeeded
    if end >= windowSize:
        windowSum -= arr[start]
        start += 1
    
    print("window sum is ", windowSum)